packages/aws-cdk-lib/aws-lambda-nodejs/lib/types.ts

```diff
@@ -437,4 +450,4 @@ export enum Charset {
    * Keep original characters without using escape sequences
    */
   UTF8 = 'utf8',
-}
+}
```

Comment:
The newline at the end of the file should be kept - not sure why this doesn't result in a linting error
Reference: https://github.com/aws/aws-cdk/blob/bf77e51c90e3da972c464430d579695163160a13/tools/%40aws-cdk/cdk-build-tools/config/eslintrc.js#L72